Corrected compile errors

Mohammed-AbdulRahman-Apryse opened the pull request:
1. Updated the UI to the variable.
2. Updated the jQuery.
3. Updated the contact sales statement.
4. Added the automatically loading of localhost.
5. Updated the dependencies versions.
6. Updated readme contents, links and structure.

Mohammed-AbdulRahman-Apryse committed:
Corrected compile errors

1. Updated the UI to the variable.
2. Updated the jQuery.
3. Updated the contact sales statement.
4. Added the automatically loading of localhost.
5. Updated the dependencies versions.
6. Updated readme contents, links and structure.

Mohammed-AbdulRahman-Apryse committed:
Adding of eslint.

1. Adding of eslint warning disables to app.js
2. Installing of eslint dependencies to the project.
3. Adding of eslint.config.mjs.

Mohammed-AbdulRahman-Apryse committed:
Updating eslint.

Updating:
1. Updated the rule to use single quote.
2. Applied the rule on the file eslint.config.mjs.

mparizeau-pdftron commented on LICENSE:
```diff
@@ -1,2 +1,2 @@
 Copyright 2018 PDFTron Systems Inc. All rights reserved.
```
Can we make this license 2024 Apryse? For example https://github.com/ApryseSDK/webviewer-ui/blob/10.12/LICENSE

mparizeau-pdftron commented on README.md:
````diff
@@ -24,15 +24,11 @@ npm install
 ```
 
-Navigate to `http://localhost:3000/`. The app will automatically reload if you change any of the source files.
+It will load `http://localhost:3000/`. The application will automatically reload when possible changes occurred to the source files.
````
```suggestion
It will load `http://localhost:3000/`. The application will automatically reload if you change any of the source files.
```
I think I like the original phrasing better here

mparizeau-pdftron commented on eslint.config.mjs:
```diff
@@ -0,0 +1,77 @@
+import js from '@eslint/js';
```
Just wondering why we added eslint to this repo? Was it on purpose? It wasn't mentioned in the PR description and doesn't really seem necessary for a sample.

Mohammed-AbdulRahman-Apryse commented:
>>> In LICENSE<https://github.com/ApryseSDK/webviewer-jquery-sample/pull/2#discussion_r1750752671>:
>>> Can we make this license 2024 Apryse? For example https://github.com/ApryseSDK/webviewer-ui/blob/10.12/LICENSE

Yes, I changed it to 2024.

----------------------------------------------------
>>> In README.md<https://github.com/ApryseSDK/webviewer-jquery-sample/pull/2#discussion_r1750753683>:
>>> I think I like the original phrasing better here
>>> It will load `http://localhost:3000/`. The application will automatically reload if you change any of the source files.

I changed it in the source.

----------------------------------------------------
>>> In eslint.config.mjs<https://github.com/ApryseSDK/webviewer-jquery-sample/pull/2#discussion_r1750755194>:
>>> +import js from ***@***.***/js';
>>> Just wondering why we added eslint to this repo? Was it on purpose? It wasn't mentioned in the PR description and doesn't really seem necessary for a sample.

The CONTRIBUTING.md<https://github.com/ApryseSDK/webviewer-jquery-sample/blob/master/CONTRIBUTING.md> -> Pull requests -> step# 4 is stating: Lint your code with `npm run lint`.
I hope this describes the addition of eslint.

So do think I should remove it from the project, and remove the step# 4 of CONTRIBUTING.md?


________________________________
From: Matt Parizeau ***@***.***>
Sent: Monday, September 9, 2024 9:47 PM
To: ApryseSDK/webviewer-jquery-sample ***@***.***>
Cc: Mohammed AbdulRahman ***@***.***>; Author ***@***.***>
Subject: Re: [ApryseSDK/webviewer-jquery-sample] Corrected compile errors (PR #2)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

@mparizeau-pdftron requested changes on this pull request.

________________________________

In LICENSE<https://github.com/ApryseSDK/webviewer-jquery-sample/pull/2#discussion_r1750752671>:

> @@ -1,2 +1,2 @@
 Copyright 2018 PDFTron Systems Inc. All rights reserved.


Can we make this license 2024 Apryse? For example https://github.com/ApryseSDK/webviewer-ui/blob/10.12/LICENSE

________________________________

In README.md<https://github.com/ApryseSDK/webviewer-jquery-sample/pull/2#discussion_r1750753683>:

> @@ -24,15 +24,11 @@ npm install
 npm start
 ```

-Navigate to `http://localhost:3000/`. The app will automatically reload if you change any of the source files.
+It will load `http://localhost:3000/`. The application will automatically reload when possible changes occurred to the source files.


⬇️ Suggested change

-It will load `http://localhost:3000/`. The application will automatically reload when possible changes occurred to the source files.
+It will load `http://localhost:3000/`. The application will automatically reload if you change any of the source files.


I think I like the original phrasing better here

________________________________

In eslint.config.mjs<https://github.com/ApryseSDK/webviewer-jquery-sample/pull/2#discussion_r1750755194>:

> @@ -0,0 +1,77 @@
+import js from ***@***.***/js';


Just wondering why we added eslint to this repo? Was it on purpose? It wasn't mentioned in the PR description and doesn't really seem necessary for a sample.

—
Reply to this email directly, view it on GitHub<https://github.com/ApryseSDK/webviewer-jquery-sample/pull/2#pullrequestreview-2290685317>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BK57WX7TU4GJBG5ZIJYYLPLZVXUM5AVCNFSM6AAAAABNSNIRM6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEOJQGY4DKMZRG4>.
You are receiving this because you authored the thread.

mparizeau-pdftron reviewed (changes requested):
Since lint is already done now it's fine we can just keep it. 

For the other changes it doesn't look like they got pushed through to the PR.

Mohammed-AbdulRahman-Apryse committed:
Requested enhancements by Matt Parizeau

1. Corrected copyright year to 2024 of License.
2. Back to the original automatic reloading text in Readme
3. Update eslint version.

mparizeau-pdftron commented on LICENSE:
```diff
@@ -1,2 +1,2 @@
-Copyright 2018 PDFTron Systems Inc. All rights reserved.
-WebViewer UI project/codebase or any derived works is only permitted in solutions with an active commercial PDFTron WebViewer license. For exact licensing terms please refer to your commercial WebViewer license. For use in other scenario, please contact sales@pdftron.com
\ No newline at end of file
+Copyright 2024 PDFTron Systems Inc. All rights reserved.
```
Can we change this to Apryse

Mohammed-AbdulRahman-Apryse committed:
Upgrade to WebViewer 10.12.0

- Upgraded WebViewer in package.json
- PDFTron -> Apryse
